### PR TITLE
Add small changes to tutorial

### PR DESCRIPTION
**Pull request by manna-harbour**
Here's some small changes to the tutorial that I would have found helpful as a new user.  Each one is a separate commit in case you'd like to choose among them.

Thanks!

### CODE REVIEW

File: keymap/tutorial.kbd [hunk 1]
```diff
@@ -344,8 +349,9 @@
 
   We have already seen other examples of modded buttons, \(, \), *, and +. There
   are no Keycodes for these buttons in KMonad, but they are buttons. They simply
-  evaluate to `(around lsft x)`. All shifted numbers have their corresponding
-  characters, the same is true for all capitals, and < > : ~ " | { } + and ?.
+  evaluate to `(around lsft x)`. All shifted numbers ( ! @ # $ % ^ & * \( \) )
+  letters (A-Z), and symbols ( ~ \_ + { } | : " < > ? ) represent themselves in
+  this way.
```
Comment (slotThe) [inline keymap/tutorial.kbd:355]: I think this new version is too US-centric, layout wise; other countries have different symbols on `S-<number>`

File: keymap/tutorial.kbd [hunk 2]
```diff
@@ -190,7 +192,9 @@
   its Keycode name, but all lower-case and with the 'Key' prefix removed. There
   are also various aliases for Keycodes starting around line 350. If you are
   trying to bind a key and there is not a 4-letter alias, please file an issue,
-  or better yet, a pull-request, and it will be added promptly.
+  or better yet, a pull-request, and it will be added promptly. Also see
+  './keymap/template/us_ansi_100.kbd' for all the keycodes used in the 104-key
+  US QWERTY layout.
 
   Also, you can consult './keymap/template/' for various input templates. Here
   we use the input-template for 'us_ansi_60.kbd'
```
Comment (slotThe) [inline keymap/tutorial.kbd:200]: The very next two lines tell the user to consult the `./keymap/template/` directory, I don't see why we need to specifically mention one single layout.

File: keymap/tutorial.kbd [hunk 3]
```diff
@@ -39,6 +39,8 @@
 
   Also, as you might have noticed, whitespace is possible anywhere.
 
+  To check for syntax errors while editing, invoke kmonad with the -d option
+  from your editor.
   -------------------------------------------------------------------------- |#
```
Comment (slotThe) [inline keymap/tutorial.kbd:44]: I would remove the "from your editor" here.

**Comment by manna-harbour** [comment]
Thanks for your review.  I've made some changes that should address your concerns.

**Comment by david-janssen** [comment]
Thanks, this is lovely

**Comment by manna-harbour** [comment]
Thanks everyone!

**Pull request merged**